go-selinux/selinux.go

```diff
@@ -62,15 +62,28 @@ func ClassIndex(class string) (int, error) {
 }
 
 // SetFileLabel sets the SELinux label for this path or returns an error.
+// follow symlinks
 func SetFileLabel(fpath string, label string) error {
 	return setFileLabel(fpath, label)
 }
 
+// SetFileLabel sets the SELinux label for this path or returns an error.
+// does not follow symlinks
+func LSetFileLabel(fpath string, label string) error {
```

Comment:
Maybe it should be named `LsetFileLabel` as other similar functions have lowercase letter after L (os.Lstat, unix.Lsetxattr).